crates/air/src/args.rs

```diff
@@ -39,3 +44,18 @@ pub(crate) struct FormatCommand {
 
 #[derive(Clone, Debug, Parser)]
 pub(crate) struct LanguageServerCommand {}
+
+/// All configuration options that can be passed "globally"
+#[derive(Debug, Default, clap::Args)]
+#[command(next_help_heading = "Global options")]
+pub(crate) struct GlobalOptions {
+    /// The log level. One of: `error`, `warn`, `info`, `debug`, or `trace`. Defaults
+    /// to `warn`.
+    #[arg(long, global = true)]
+    pub(crate) log_level: Option<logging::LogLevel>,
+
+    /// Disable colored output. To turn colored output off, either set this option or set
+    /// the environment variable `NO_COLOR` to any non-zero value.
+    #[arg(long, global = true)]
+    pub(crate) no_color: bool,
```

Comment:
I'd maybe move this to an advanced section in the future. Since we don't have many options it'd be weird to do this now.